app/models/generic_event/move_lockout.rb

```diff
@@ -27,17 +28,6 @@ def from_location
       Location.find_by(id: from_location_id)
     end
 
-    def for_feed
-      super.tap do |common_feed_attributes|
-        common_feed_attributes['details'] = from_location.for_feed(prefix: 'from')
-        common_feed_attributes['details'].merge!(
-          'authorised_at' => authorised_at,
```

Comment:
These attributes don't get replaced by the new shared module - were they removed intentionally?

Reply:
They come in for free since we're simply deleting the location key rather than building the hash from scratch (see line 32 where we're assigning the details anew from the result of calling the `from_location.for_feed`

Reply:
Coolio 👍